content/zh/docs/multicluster-management/unbind-cluster.md

```diff
@@ -19,11 +19,11 @@ weight: 5500
 
 2. 在**集群管理**页面，请点击要从中央控制平面移除的集群。
 
-   ![集群管理](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/cluster-management.PNG)
+   ![集群管理](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/cluster-management.png)
 
 3. 在**集群设置**下的**基本信息**页面，请选择**我确定要执行解绑集群的操作**，然后点击**解除绑定**。
 
-   ![解绑集群](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/unbind-cluster.PNG)
+   ![解绑集群](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/unbind-cluster.png)
```

Comment:
Double check in your local build to see if this image is visible.

Reply:
All fixed.